test/api/fromUrlStatus.spec.js

```diff
@@ -0,0 +1,30 @@
+import fromUrlStatus from '../../src/api/fromUrlStatus'
+import * as factory from '../_fixtureFactory'
+
+describe('API - from url status', () => {
+  it('should return info about file uploaded from url', async() => {
+    const data = await fromUrlStatus(factory.token('valid'))
+
+    expect(data.uuid).toBeTruthy()
```

Comment:
from url status returns response with upload info, see review comment in fromUrlStatus